src/components/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,5 @@
+import { SkeletonWalletMenuButtonStyled } from './Skeleton.style';
+
+export const SkeletonWalletMenuButton = () => {
```

Comment:
no need for this component. You can directly use it `SkeletonWalletMenuButtonStyled` in the other file